<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QDialog" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(223, 95, 159, 200), stop:0.513812 rgba(224, 212, 107, 200), stop:1 rgba(103, 188, 228, 200));</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>600</width>
     <height>850</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);

border-radius : 10px;</string>
   </property>
   <widget class="QPushButton" name="makescedule_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>802</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(54, 54, 54);
background-color: rgba(255, 255, 255, 240);

font-family : 'Dosis';
font-weight : bold;
font-size : 10pt;

border-radius : 8px;</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QComboBox" name="locateselect_combobox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>800</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 240);

font-weight : 400;
font-family : 'Dosis';
font-size : 11pt;

border-radius : 4px;</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="time_table">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>19</y>
      <width>540</width>
      <height>771</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgba(64, 255, 240, 150);
selection-background-color: rgba(64, 255, 240, 150);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::DotLine</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>67</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>34</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>09:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>09:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>21:00</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>21:30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>22:00</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>S</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>M</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>T</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>W</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>T</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>F</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>S</string>
     </property>
    </column>
    <item row="5" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="where_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>800</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

font-weight : 600;
font-family : 'Dosis';
font-size : 14pt;</string>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>5</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 240);

border-radius : 10px;</string>
   </property>
   <widget class="QPushButton" name="gotologin_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>2</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);

font-weight : 600;
font-family : 'Dosis';
font-size : 10pt;</string>
    </property>
    <property name="text">
     <string>‚Üê Back to Login</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
